tests/beacon/test_helpers.py

```diff
@@ -61,6 +63,10 @@
 )
 
 
+class UnreachableCodePathError(Exception):
```

Comment:
We've tended to just use `Exception` for this as it's more of a visual indication that the code path can't be reached and were we to encounter the actual exception the fix should be quite straight forward.

Reply:
sounds good -- was electing to be as explicit as possible but what you say makes sense